sql_generators/active_users/templates/mobile_deletion_request_query.sql

```diff
@@ -0,0 +1,241 @@
+    BETWEEN @start_date
+    AND @end_date
+),
+search_clients AS (
+  SELECT
+    client_id,
+    submission_date,
+    ad_click,
+    organic,
+    search_count,
+    search_with_ads
+  FROM
+    `moz-fx-data-shared-prod.search_derived.mobile_search_clients_daily_v1`
+  WHERE
+    submission_date = @submission_date
+    AND app_name = '{{ app_value }}'
+),
+search_metrics AS (
+  SELECT
+    baseline.client_id,
+    baseline.submission_date,
+    SUM(ad_click) AS ad_clicks,
+    SUM(organic) AS organic_search_count,
+    SUM(search_count) AS search_count,
+    SUM(search_with_ads) AS search_with_ads
+  FROM
+    baseline
+  LEFT JOIN
+    search_clients s
+  ON
+    baseline.client_id = s.client_id
+    AND baseline.submission_date = s.submission_date
+  GROUP BY
+    client_id,
+    submission_date
+),
+baseline_with_searches AS (
+  SELECT
+    baseline.client_id,
+    CASE
+      WHEN BIT_COUNT(days_seen_bits)
+        BETWEEN 1
+        AND 6
+        THEN 'infrequent_user'
+      WHEN BIT_COUNT(days_seen_bits)
+        BETWEEN 7
+        AND 13
+        THEN 'casual_user'
+      WHEN BIT_COUNT(days_seen_bits)
+        BETWEEN 14
+        AND 20
+        THEN 'regular_user'
+      WHEN BIT_COUNT(days_seen_bits) >= 21
+        THEN 'core_user'
+      ELSE 'other'
+    END AS activity_segment,
+    baseline.app_name,
+    baseline.app_display_version AS app_version,
+    baseline.normalized_channel,
+    IFNULL(country, '??') country,
+    baseline.city,
+    baseline.days_seen_bits,
+    baseline.days_created_profile_bits,
+    DATE_DIFF(baseline.submission_date, baseline.first_seen_date, DAY) AS days_since_first_seen,
+    baseline.device_model,
+    baseline.isp,
+    baseline.is_new_profile,
+    baseline.locale,
+    baseline.first_seen_date,
+    baseline.days_since_seen,
+    baseline.normalized_os,
+    baseline.normalized_os_version,
+    COALESCE(
+      SAFE_CAST(NULLIF(SPLIT(baseline.normalized_os_version, ".")[SAFE_OFFSET(0)], "") AS INTEGER),
+      0
+    ) AS os_version_major,
+    COALESCE(
+      SAFE_CAST(NULLIF(SPLIT(baseline.normalized_os_version, ".")[SAFE_OFFSET(1)], "") AS INTEGER),
+      0
+    ) AS os_version_minor,
+    COALESCE(
+      SAFE_CAST(NULLIF(SPLIT(baseline.normalized_os_version, ".")[SAFE_OFFSET(2)], "") AS INTEGER),
+      0
+    ) AS os_version_patch,
+    baseline.durations,
+    baseline.submission_date,
+    baseline.uri_count,
+    baseline.is_default_browser,
+    baseline.distribution_id,
+    CAST(NULL AS string) AS attribution_content,
+    CAST(NULL AS string) AS attribution_source,
+    CAST(NULL AS string) AS attribution_medium,
+    CAST(NULL AS string) AS attribution_campaign,
+    CAST(NULL AS string) AS attribution_experiment,
+    CAST(NULL AS string) AS attribution_variation,
+    search.ad_clicks,
+    search.organic_search_count,
+    search.search_count,
+    search.search_with_ads,
+    NULL AS active_hours_sum,
+    @partition_date AS partition_date
```

Comment:
rather than define `partition_date` here and pass it through each query, it should just be defined in the final query:
```suggestion
```